src/ui/settings/page.js

```diff
@@ -3,25 +3,26 @@
 import { jsx, Styled } from 'theme-ui';
 
 import { useTranslation } from 'react-i18next';
-import { Container } from '@theme-ui/components';
+import { Box } from '@theme-ui/components';
 
 import LanguageSettings from './language';
 import OpencastSettings from './opencast';
 
 
-const SettingsPage = ({ settingsManager }) => {
+const SettingsPage = ({ settingsManager, hasRecording }) => {
+  console.log(hasRecording);
   const { t } = useTranslation();
 
   return (
-    <Container sx={{ p: 3 }}>
+    <Box sx={{ width: '100%', maxWidth: 960, mx: 'auto', p: 3 }}>
```

Comment:
Is this a regression? Shouldn't this be a Container?

Reply:
That's actually intended as `Container` is very wide. Looks pretty bad IMO when input boxes are 1400px or so wide. Using `Container` and still specifying a `maxWidth` doesn't work.